src/error/ParseError.js

```diff
@@ -21,7 +21,7 @@ class ParseError extends Error {
     endColumn = 0,
     severity = 'Error',
   ) {
-    super(message);
+    super(message || 'Error happened when trying to parse.');
```

Comment:
`|| ...` not needed (see L.17)

Reply:
Not agree, because user can put `null`, default in parameters are only used for `undefined`